x-pack/platform/plugins/private/reporting/public/management/components/scheduled_report_flyout_content.tsx

```diff
@@ -0,0 +1,407 @@
+    () => (
+      <EuiLink href={http.basePath.prepend(REPORTING_MANAGEMENT_SCHEDULES)}>
+        {i18n.REPORTING_PAGE_LINK_TEXT}
+      </EuiLink>
+    ),
+    [http.basePath]
+  );
+  const { mutateAsync: scheduleReport, isLoading: isScheduleExportLoading } = useScheduleReport({
+    http,
+  });
+  const { defaultTimezone } = useDefaultTimezone();
+  const now = useMemo(() => moment().tz(defaultTimezone), [defaultTimezone]);
+  const defaultStartDateValue = useMemo(() => now.toISOString(), [now]);
+  const schema = useMemo(
+    () =>
+      getScheduledReportFormSchema(
+        validateEmailAddresses,
+        availableReportTypes
+      ) as FormSchema<FormData>,
+    [availableReportTypes, validateEmailAddresses]
+  );
+  const recurring = true;
+  const startDate = defaultStartDateValue;
+  const timezone = defaultTimezone;
+  const { form } = useForm<FormData>({
+    defaultValue: scheduledReport,
+    options: { stripEmptyFields: true },
+    schema,
+    onSubmit: async (formData) => {
+      try {
+        const {
+          title,
+          reportTypeId,
+          recurringSchedule,
+          optimizedForPrinting,
+          sendByEmail,
+          emailRecipients,
+        } = formData;
+        // Remove start date since it's not supported for now
+        const { dtstart, ...rrule } = convertToRRule({
+          startDate: now,
+          timezone,
+          recurringSchedule,
+          includeTime: true,
+        });
+        await scheduleReport({
+          reportTypeId,
+          jobParams: getReportParams({
+            apiClient,
+            // The assertion at the top of the component ensures these are defined when scheduling
+            sharingData: sharingData!,
+            objectType: objectType!,
+            title,
+            reportTypeId,
+            ...(reportTypeId === 'printablePdfV2' ? { optimizedForPrinting } : {}),
+          }),
+          schedule: { rrule: rrule as Rrule },
+          notification: sendByEmail ? { email: { to: emailRecipients } } : undefined,
+        });
+        toasts.addSuccess({
+          title: i18n.SCHEDULED_REPORT_FORM_SUCCESS_TOAST_TITLE,
+          text: mountReactNode(
+            <>
+              {i18n.SCHEDULED_REPORT_FORM_SUCCESS_TOAST_MESSAGE} {reportingPageLink}.
+            </>
+          ),
+        });
+      } catch (error) {
+        // eslint-disable-next-line no-console
+        console.error(error);
+        toasts.addError(error, {
+          title: i18n.SCHEDULED_REPORT_FORM_FAILURE_TOAST_TITLE,
+          toastMessage: i18n.SCHEDULED_REPORT_FORM_FAILURE_TOAST_MESSAGE,
+        });
+        // Forward error to signal whether to close the flyout or not
+        throw error;
+      }
+    },
+  });
+  const [{ reportTypeId, sendByEmail }] = useFormData<FormData>({
+    form,
+    watch: ['reportTypeId', 'sendByEmail'],
+  });
+
+  useEffect(() => {
+    if (!hasManageReportingPrivilege && userProfile?.user.email) {
```

Comment:
I had to do this imperatively since the form `defaultValue` doesn't seem to be reactive